Bottom: 02615efe2058032a04219c5cd8d561302254e504
Top:    f19a8b2ce69c643de5d0f18daa34e98ee942afed
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-25 00:15:54 -0700

Refresh of riostream

---

diff --git a/man/rsocket.7 b/man/rsocket.7
index dc3f9ab..bc5bb10 100644
--- a/man/rsocket.7
+++ b/man/rsocket.7
@@ -7,8 +7,8 @@ rsocket \- RDMA socket API
 RDMA socket API and protocol
 .SH "NOTES"
 Rsockets is a protocol over RDMA that supports a socket-level API
-for applications.  rsocket APIs are intended to match the behavior
-of corresponding socket calls, except where noted.  rsocket
+for applications.  Rsocket APIs are intended to match the behavior
+of corresponding socket calls, except where noted.  Rsocket
 functions match the name and function signature of socket calls,
 with the exception that all function calls are prefixed with an 'r'.
 .P
